packages/now-cli/test/helpers/prepare.js

```diff
@@ -500,12 +500,7 @@ CMD ["node", "index.js"]`,
       }),
     },
     'project-link': {
-      'pages/index.js': 'export default () => <div><h1>Now CLI test</h1></div>',
-      'package.json': JSON.stringify({
-        dependencies: {
-          gatsby: 'latest',
-        },
-      }),
```

Comment:
I think you should leave gatbsy here to prove that the project settings will override the defaults, right?

Reply:
The problem was that gatsby has a default dev command, which we want to avoid so that the build command is run. I think it's fine to remove gatsby since the build command gets overwritten and executed in both prod and dev, which is what we're testing for here.